client/src/app/pages/dashboard/services/tasks.service.ts

```diff
@@ -0,0 +1,122 @@
+import { Injectable } from '@angular/core';
+import { AddTask, EditTask, StatusValue, Task } from '../models/models.tasks';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class TasksService {
+  #currentIDNumber = 5;
+  #indexCompleted = 0;
+  #indexInProgress = 0;
+
+  #tasksMany: Task[] = [
```

Comment:
DO we really need this data? And again name is very bad

Reply:
No, we do not necessarily need it. I will delete it